File: src/main/webapp/js/common/controller/HomeController.js
'use strict';
(function () {
    angular.module('L2F.Common').controller('HomeController', ['$http', HomeController]);

    function HomeController($http) {
        var home = this;

        $http.get('news').success(function (response) {
            home.news = response.news;
        });

    }
})();

File: src/main/java/com/github/vendigo/l2f/home/HomeController.java
package com.github.vendigo.l2f.home;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;

@RestController
@RequestMapping("/api")
public class HomeController {
    @RequestMapping(value = "/news", method = RequestMethod.GET)
    public Map<String, String> news() {
        Map<String, String> response = new HashMap<>();
        response.put("news", "Latest news!");
        return response;
    }
}
